frontend/src/component/feature/FeatureView/FeatureMetrics/FeatureExposureMetrics.test.tsx

```diff
@@ -0,0 +1,96 @@
+import { vi, beforeEach, describe, it, expect } from 'vitest';
+import type React from 'react';
+import { render } from 'utils/testRenderer';
+import { screen } from '@testing-library/react';
+
+type Env = {
+    name: string;
+    type: 'production' | 'development' | 'staging';
+};
+let ENVS: Env[] = [];
+let PROJECT_ID = 'p';
+let FEATURE_ID = 'f';
+
+vi.mock('hooks/useRequiredPathParam', () => ({
+    useRequiredPathParam: (k: string) =>
+        k === 'projectId' ? PROJECT_ID : k === 'featureId' ? FEATURE_ID : '',
+}));
+
+vi.mock('hooks/api/getters/useFeature/useFeature', () => ({
+    useFeature: () => ({
+        feature: { name: FEATURE_ID, project: PROJECT_ID, environments: ENVS },
+        loading: false,
+        error: null,
+        refetchFeature: vi.fn(),
+    }),
+}));
+
+vi.mock('hooks/api/getters/useFeatureMetrics/useFeatureMetrics', () => ({
+    useFeatureMetricsRaw: () => ({
+        featureMetrics: [],
+        loading: false,
+        error: null,
+    }),
+}));
```

Comment:
As previously stated, please use testServerRoute so we do not need to use `vi.mock.` Lets try to completely get rid of `vi.mock` in this test. https://github.com/Unleash/unleash/pull/10891#discussion_r2480964710